src/js/controllers/homeController.js

```diff
@@ -1,23 +1,38 @@
 import Home from '../models/home';
 import {elements} from '../views/base';
 import * as homeView from '../views/homeView';
+import * as oppView from '../views/oppView';
 
 
 export const controlHome = async (state) => {
     //1) Get contact list(array)
     //console.log(userId);
     await state.home.getHomeData();
     await state.opp.getOppData();
+    if (state.user.isSenior){
+        console.log("senior present");
+        await state.home.seniorGetOpps(state.user.uid);
+    } else {
+        await state.home.getMyOpps(state.user.uid);
+    }
+    
     state.user.name = state.home.profile.name;
 
+    console.log(state.home.seniorOpps)
     //2) Prepare UI(optional)
-    homeView.clearProfile();
+    homeView.clearProfile(state.user.isSenior);
     
     //3) Render UI
     homeView.renderProfile(state.home.profile);
     homeView.renderAbout(state.home.profile);
     homeView.renderExps(state.home.profile.experience);
-    homeView.renderOpps(state.home.profile,state.opp.opps);
+    if (state.user.isSenior){
+        oppView.renderOpps(state.home.seniorOpps);
+        //console.log(state.home.seniorOpps[0].data());
+    } else {
+        homeView.renderOpps(state.home.myOpps,state.opp.opps);
+    }
+
```

Comment:
Instead of making so many function calls, can we just call one function like `homeView.renderHome(...)` which does whatever it needs to render the whole page?